create_db.py

```diff
@@ -38,9 +38,9 @@ def create_default_user(email, password):
     parser = argparse.ArgumentParser()
     parser.add_argument("email", nargs='?', help="The email for super_admin.", default='')
     parser.add_argument("password", nargs='?', help="The password for super_admin.", default='')
-    parser.parse_args()
-    email = parser.email if hasattr(parser, "email") else ''
-    password = parser.password if hasattr(parser, "password") else ''
+    parse = parser.parse_args()
+    email = parse.email if hasattr(parse, "email") else ''
```

Comment:
We can use the `getattr` BIF:

`email = getattr(parse, 'email', '')`

Reply:
:+1:

Reply:
This can be simply replaced with `email = parser.get('email')` which will return None if not found.

Reply:
@poush If he doesn't want empty strings as default values for specific reasons, then `parser.get()` call seems to be the right way to go.